avocado/utils/path.py

```diff
@@ -83,6 +83,7 @@ def find_command(cmd, default=None):
                     in the standard directories.
     :raise: :class:`avocado.utils.path.CmdNotFoundError` in case the
             command was not found and no default was given.
+    :rtype: Boolean. The default was set with path or to False
```

Comment:
Hi @anascko , thanks for this. It is always good to clarify the returns.

But, on line 99 we have a `return os.path.abspath(cmd_path)`. This is not a boolean, right?

So, I might be wrong but this is returning: "A string with a normalized absolutized version of command or the default if the command is not found.", right?

Reply:
Thanks for the review :) 
Oh, sorry :(  When I tested I didn't check the outputs and use it in skipUnless, and of course it passe, because I was expecting the boolean False.
I'll promise to look in code before doing something :(
Yes, it reverts the binary path if exists or value from the default if it not